cns/service/main.go

```diff
@@ -312,8 +313,12 @@ func printVersion() {
 	fmt.Printf("Version %v\n", version)
 }
 
+type NodeInquirer interface {
```

Comment:
I'm imagining this with a top hat and monocle. I would have gone with "Interrogator" 😅

Reply:
I like `Interrogator`. I wanted to separate out the read ops from the write ops. `NodeReader` sounded... weird and misleading. `Interrogator` does say what it does what it says on the tin... it interrogates nodes for information. I guess that means the writing side should be `NodeInceptor` 😆

Reply:
Updated in 9381406